etc/org.opencastproject.security.lti.LtiLaunchAuthenticationHandler.cfg

```
@@ -48,12 +48,23 @@
 # Default: no blacklisted users
 #lti.blacklist.user.1=
 
-# Determines whether a JpaUserReference should be created on LTI User Login.
-#
-# This persists the LTI Users in the database, giving them the ability to create long running tasks like ingesting a video.
-#
-# Default: true
-#lti.create_jpa_user_reference=true
+# Determines if a user reference should be persisted in the database on LTI user logins.
+# Persisting references gives the users the ability to create long running tasks like ingesting a video.
+# The value can be a list of LTI roles identifying users to be persisted or the special value * causing all users to be persisted.
+# The value is not case sensitive.
+#
+# Examples:
+#  - Persist only instructors:
+#    lti.create_jpa_user_reference = instructor
+#  - Persist only instructors and administrators:
+#    lti.create_jpa_user_reference.roles = instructor, administrator
+#  - Persist all users:
+#    lti.create_jpa_user_reference.roles = *
+#  - No persistence:
+#    lti.create_jpa_user_reference.roles =
+#
+# Default: *
+lti.create_jpa_user_reference.roles = *
```

Setting a default should be unnecessary:

```suggestion
#lti.create_jpa_user_reference.roles = *
```